### PR TITLE
TCD-2811: Replace wording methods with endpoints

### PR DESCRIPTION
**Pull request by LoryApiFortress**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by LoryApiFortress** [comment]
@aliciakrebs @lysannep please review. Thanks

**Review by aliciakrebs — APPROVED** [review]

File: docs/dev/api/accounts.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ The Accounts API exposes the following methods related to individual and team ac
```
Comment (lysannep) [inline docs/dev/api/accounts.md:12]: The description above (on line 11) could be updated as well:

**from:** _The Accounts API exposes the following **methods** related to individual and team account configuration and monitoring._  
**to:** _The Accounts API exposes the following **endpoints** related to individual and team account configuration and monitoring._

File: docs/dev/api/accounts.md [hunk 2]
```diff
@@ -1501,12 +1501,14 @@ curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
 <p/>
 
 Returns details about the current in-use virtual machines and real devices along with the maximum allowed values.
-  
+
 :::note Real Devices
 At this time, the current usage for real devices is not accurately returned in the response. As a workaround, use the following endpoint:
```
Comment (lysannep) [inline docs/dev/api/accounts.md:1506]:
```suggestion
:::note Real Devices

At this time, the current usage for real devices is not accurately returned in the response. As a workaround, use the following endpoint:
```

**Comment by LoryApiFortress** [comment]
@lysannep changes applied :)

**Review by lysannep — APPROVED** [review]

**Pull request merged**